draft-ietf-ppm-dap.md

```diff
@@ -1295,7 +1296,13 @@ An aggregation job can be thought of as having three phases:
 - Completion: Finish the aggregate flow, yielding an output share corresponding
   to each report share in the aggregation job.
```

Comment:
This is still weird.  If completing the aggregation job doesn't involve any aggregation, then it doesn't seem to be an aggregation job.  Maybe it's a preparation job, or a sub-aggregation job, or a pre-aggregation job.  Or maybe this phase isn't "completion"; maybe it's "collection".

Reply:
I agree with this: while implementations _can_ (and, in practice, often _do_) aggregate the output shares into a running-total aggregate share at the end of processing an aggregation job, this is not required. And depending on the VDAF/batch mode in use, this aggregation may not be possible at this time.

Since performing aggregation at this stage is an optional performance optimization, I agree that "aggregation job" is not the best name.

I would support "preparation job", since the process that is occurring is most correctly called "VDAF preparation". I think "sub-aggregation job"/"pre-aggregation job" are questionable for the same reason that "aggregation job" is.

Reply:
"Collection" is a term used for the "collection sub-protocol", in which the batch is selected and the aggregate shares are computed. Let's not overload it.

I'm open to renaming, but the terms  "aggregation job" and "collection job" are pretty baked in at this point, into code and into the way we talk about the protocol. In any case, let's please not let this discussion block this PR, as I think it's a positive improvement in any case. I"ll even volunteer to file an issue and lead the discussion.

Reply:
FWIW, I agree that the renaming process would be somewhat painful in terms of the work it induces in existing implementations. I'm not sure that justifies avoiding a rename, however.

Reply:
Again, I'm not rejecting to renaming. I'm just trying to make incremental progress on the issues.